.eslintrc

```diff
@@ -35,6 +35,18 @@
     "es6": true
   },
 
+  "globals": {
```

Comment:
This would allow for us to use jasmine globals in the main codebase; which I don't think is intentional.